pkg/primitives/statistics.go

```diff
@@ -60,16 +60,16 @@ func NewStatistics(total gridtypes.Capacity, storage provision.Storage, reserved
 	}
 }
 
-// Get all used capacity from storage + reserved
-func (s *Statistics) active() (gridtypes.Capacity, error) {
-	cap, _, err := s.storage.Capacity()
+// Get all used capacity from storage + reserved / deployments count and workloads count
+func (s *Statistics) active() (gridtypes.Capacity, int, int, error) {
```

Comment:
returning an `int` and `int` is confusing because it's not clear what they are without given names. Also the return argument list become too long imho 🤷🏼‍♂️ 

there is another comment about the return of Storage.Capacity() once it's done, this function can just get that, make sure to add the reserved to the capacity then return the same structure